src/sagemaker/estimator.py

```diff
@@ -639,18 +640,18 @@ def deploy(
                 model completes (default: True).
             model_name (str): Name to use for creating an Amazon SageMaker
                 model. If not specified, the name of the training job is used.
-            tags(List[dict[str, str]]): Optional. The list of tags to attach to this specific
-                endpoint. Example:
-                >>> tags = [{'Key': 'tagname', 'Value': 'tagvalue'}]
-                For more information about tags, see
-                https://boto3.amazonaws.com/v1/documentation\
-                /api/latest/reference/services/sagemaker.html#SageMaker.Client.add_tags
             kms_key (str): The ARN of the KMS key that is used to encrypt the
                 data on the storage volume attached to the instance hosting the
                 endpoint.
             data_capture_config (sagemaker.model_monitor.DataCaptureConfig): Specifies
                 configuration related to Endpoint data capture for use with
                 Amazon SageMaker Model Monitoring. Default: None.
+            tags(List[dict[str, str]]): Optional. The list of tags to attach to this specific
+                endpoint. Example:
+                >>> tags = [{'Key': 'tagname', 'Value': 'tagvalue'}]
```

Comment:
do you need newlines around the ">>>" line for the formatting to work out?

Reply:
Looks like the >>> doesn't work as intended as shown here: https://sagemaker.readthedocs.io/en/v1.45.0/analytics.html#sagemaker.analytics.HyperparameterTuningJobAnalytics.tuning_ranges

Will fix in another PR D: